ambry-api/src/main/java/com/github/ambry/account/Dataset.java

```diff
@@ -61,8 +61,8 @@ public class Dataset {
   static final String CONTAINER_NAME_KEY = "containerName";
   static final String DATASET_NAME_KEY = "datasetName";
   static final String JSON_VERSION_SCHEMA_KEY = "versionSchema";
-  static final String JSON_EXPIRATION_TIME_KEY = "expirationTimeMs";
   static final String JSON_RETENTION_COUNT_KEY = "retentionCount";
+  static final String JSON_RETENTION_TIME_KEY = "retentionTime";
```

Comment:
Looks like "retentionTime" is in seconds. Can we rename it to include the time unit?

Reply:
Sure, updated.